src/components/Label/Label.tsx

```diff
@@ -63,6 +63,23 @@ export const Label: FC<LabelProps> = ({
         styles.fieldLabelIconButton,
         sizeClassNames,
     ]);
+    const defaultIconButtonStyles = (
+        labelIconButtonProps: LabelIconButtonProps
+    ): React.CSSProperties => {
+        let styles: React.CSSProperties = null;
+        // If any of the defaults are overriden don't style the button height or padding.
+        if (
+            !labelIconButtonProps.size &&
+            !labelIconButtonProps.shape &&
+            !labelIconButtonProps.text
+        ) {
+            styles = {
```

Comment:
cant this be done via a class ?

Reply:
The button module prevents it as it has priority.